src/views/monitor/components/left/left-two.vue

```diff
@@ -15,10 +15,10 @@
         <table>
           <tbody>
             <tr v-for="item in 8">
-              <td>产品名</td>
-              <td>产品编号产品编号产品编号产品编号</td>
-              <td>位置</td>
-              <td>时间</td>
+              <td>摄像头{{ item }}</td>
+              <td>菇房{{ item }}</td>
```

Comment:
```suggestion
              <td>摄像头 {{ item }}</td>
              <td>菇房 {{ item }}</td>
```

数字和文字之间增加空格使格式看起来更加便于阅读

Reply:
临时格式，用户截图